content/jobs/tanks/dark-knight/advanced-guide.md

```diff
@@ -114,34 +123,57 @@ Living Shadow's base strength is calculated differently to the player. In partic
 
-`baseStrength + strengthProvidedByGear + 2`
+\[
+\begin{aligned}
+\rm{baseStrength} + \rm{strengthProvidedByGear} + 2
+\end{aligned}
+\]
 
-Living Shadow's weapon damage multplier is calculated differently to the player. In particular:
+Living Shadow's weapon damage multiplier is calculated differently to the player. In particular:
 
-- While players would use a weapon damage multplier of `115`, Living Shadow uses a weapon damage multiplier of `100`
+- While players would use a weapon damage multiplier of \(115\), Living Shadow uses a slightly lower weapon damage multiplier of \(100\)
 
 - This is typical for all 'pet actions', and generally true for any damage that does not come 'from' the player
 
 This changes the weapon damage multiplier formula from the following:
 
-`floor(baseStrength * 100 / 1000 + weaponDamage) / 100`
+\[
+\begin{aligned}
+\frac{\left \lfloor{\frac{\rm{baseStrength} \times 115}{1000} + \rm{weaponDamage}}\right \rfloor}{100}
+\end{aligned}
+\]
+
+To the following (where \(\rm{baseStrength}\) is \(440\) at level 100):
 
-To the following (where `baseStrength` is 440 at level 100):
+\[
+\begin{aligned}
+\frac{\left \lfloor{\frac{\rm{baseStrength} \times 100}{1000} + \rm{weaponDamage}}\right \rfloor}{100}
+\end{aligned}
+\]
 
-`floor(baseStrength * 115 / 1000 + weaponDamage) / 100`
 
 Living Shadow's main stat multiplier is calculated differently to the player. In particular:
 
-- The Tank Mastery trait means that tanks have a lower main stat power modifier than other jobs. The modifier varies by level, but at level 100, it is `190` for tanks, and `237` for all other roles (meaning tanks have ~20% lower main stat scaling than other roles). Living Shadow uses the 'other' scaling
+- The Tank Mastery trait means that tanks have a lower main stat power modifier than other jobs. The modifier varies by level, but at level 100, it is \(190\) for tanks, and \(237\) for all other roles (meaning tanks have ~20% lower main stat scaling than other roles). Living Shadow uses the 'other' scaling
+
+- This means that Living Shadow gains more damage scaling (about 25% higher) from strength than other abilities on Dark Knight
+
+At level 100 (where the main stat multiplier for tanks is \(190\), and other jobs is \(237\)), this changes the main stat multiplier formula from the following (where \(\rm{baseStrength}\) is \(440\) at level 100):
 
-- This means Living Shadow gains more damage scaling (about 25% higher) from strength than normal abilities on Dark Knight
 
-At level 100 (where the main stat multiplier for tanks is `190`, and other jobs is `237`), this changes the main stat multiplier formula from the following (where `baseStrength` is `440` at level 100):
+\[
+\begin{aligned}
+\frac{\left \lfloor{\frac{190 \times \left(\rm{totalStrength} - \rm{baseStrength}\right)}{\rm{baseStrength}}}\right \rfloor + 100}{100}
+\end{aligned}
+\]
```

Comment:
Aside, not for change right now: More broadly, (especially when there's more of this information in the guide), we should consider if it makes more sense to present this as:
```
1 + \frac{\left\lfloor \frac{190 \times \left(\rm{totalStrength} - \rm{baseStrength}\right)}{\rm{baseStrength}} \right\rfloor}{100}
```
instead --- I think this makes some things clearer (e.g., it's always a 1.xxx), but the stacked fraction+floor also looks a bit more messy. Maybe it becomes worth just defining some (neat) notation for "truncate the result of this computation to n decimal places", so that it can become more like:
```
\mathsf{truncate}\left(\frac{190}{100} \times \frac{\left( \rm{totalStrength} - \rm{baseStrength} \right)}{\rm{baseStrength}}, 2\right)
```
(the function notation is probably not ideal, here, but should get across the idea).